### PR TITLE
Adding Congressional District Data

### PR DESCRIPTION
**Pull request by VincentLa**
**1. Brief Summary of what this PR accomplishes (140 characters or less. If you find trouble describing what you are doing in this length, consider breaking the PR into multiple ones.)**

Hits Google Civic API to add congressional district data to our database

**2. Link to Trello Ticket**

https://trello.com/c/aKvxFeZF/36-get-congressional-districts-and-counties-for-borrower-data-this-is-to-identify-county-and-congressional-district-for-each-addres

**3. More detailed description and other questions to address in code review**

We use Google Civic API to add congressional district data to our database. Note that you will have to have an environment variable `GOOGLEAPI` to run this code. See https://developers.google.com/civic-information/ for more information about Google Civic API

Also added an example of a test for the python function that runs the API call.

**4. Remember to tag reviewers!**
@makfan64

### CODE REVIEW

File: pipeline/pipeline_tasks/queries/stg_analytics/00_01_04_sba_sfdo_all.sql [hunk 1]
```diff
@@ -103,7 +104,8 @@ select
   sba_sfdo.revolver_status,
   api_calls.yelp_rating,
   api_calls.yelp_total_reviews,
-  api_calls.yelp_url
+  api_calls.yelp_url,
+  api_calls.civic_dist
```
Comment (VincentLa) [inline pipeline/pipeline_tasks/queries/stg_analytics/00_01_04_sba_sfdo_all.sql]: @makfan64 can this be called `congressional_district`?

Comment (makfan64) [reply pipeline/pipeline_tasks/queries/stg_analytics/00_01_04_sba_sfdo_all.sql]: @VincentLa I saw another field with that name and I couldn't figure how it was used in the time I had. I agree that would be a better game.

File: pipeline/pipeline_tasks/api_calls/congressional_districts.py [hunk 2]
```diff
@@ -1 +1,56 @@
...
+      df: DataFrame with business information to request congressional districts
+    """
+
+    url = 'https://www.googleapis.com/civicinfo/v2/representatives'
+
+    if df is None or df.empty:
+        print("Incoming dataframe is empty.")
+        return
+
+    df['cong_dist'] = None
```
Comment (VincentLa14) [inline pipeline/pipeline_tasks/api_calls/congressional_districts.py]: Let's rename this to `congressional_district_google_civic`? and then we can compare to `congressional_district`

Comment (makfan64) [reply pipeline/pipeline_tasks/api_calls/congressional_districts.py]: Done and pushed.

**Comment by makfan64** [comment]
I think we should close this as PR 72 addresses this and all the other API calls.

**Pull request closed without merging**